src/common/hooks/useCommonActions.ts

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
@beganovich Yeah, actually the reason this file is in the global `hooks` folder is that I envisioned having actions for all entities in this file, including `default` actions hook and `all` actions hook. Perhaps what may have caused confusion here is that we currently have default and all actions only for the `invoice` entity. However, this is because we are currently implementing this feature exclusively for that specific entity. When we decide to implement it for other entities, I will simply add additional actions here. I hope that you agree.